packages/components/src/pages/install-guide/EmptyState.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <QuickstartLayout>
+    <section>
+      <header>
+        <h1>No projects found</h1>
+      </header>
+      <main>
+        <article class="empty-state" v-if="projects.length == 0">
```

Comment:
There's a lot of double equality operators (`==`) in here. We typically use triple equality (`===`) across the board as it's more accurate and doesn't suffer from weird JavaScript-y behaviors. Can we make sure to move to triple equals before this gets merged?

Reply:
Sure thing!